Lib/fontTools/ttLib/tables/_c_v_t.py

```diff
@@ -47,3 +48,34 @@ def __setitem__(self, index, value):
 
 	def __delitem__(self, index):
 		del self.values[index]
+
+
+class CVTValues(object):
+	""" This is used in varLib for calculating control value deltas"""
+
+	def __init__(self, values):
+		self.values = list(values)
+
+	def __getitem__(self, index):
+		return self.values[index]
+
+	def __len__(self):
+		return len(self.values)
+
+	def __repr__(self):
+		return u"CVTValues(%s)" % self.values
```

Comment:
`__repr__` usually return `str` objects, not `unicode`. It's ascii only anyway

Reply:
Fixed.